packages/desktop-client/src/components/mobile/accounts/Accounts.tsx

```diff
@@ -87,23 +112,26 @@ function AccountCard({
             alignItems: 'center',
           }}
         >
-          {account.bankId && (
-            <View
-              style={{
-                backgroundColor: pending
-                  ? theme.sidebarItemBackgroundPending
-                  : failed
-                    ? theme.sidebarItemBackgroundFailed
-                    : theme.sidebarItemBackgroundPositive,
-                marginRight: '8px',
-                width: 8,
-                flexShrink: 0,
-                height: 8,
-                borderRadius: 8,
-                opacity: connected ? 1 : 0,
-              }}
-            />
-          )}
+          {
+            /* TODO: Should bankId be part of the AccountEntity type? */
```

Comment:
I think so along with `bankName`, they'd need to be optional though

Looks like getAccounts brings them in as:
https://github.com/actualbudget/actual/blob/022b9b76b1fb45b14faf297c2480a993ff96fd24/packages/loot-core/src/server/db/index.ts#L619-L626

Reply:
I think we should handle this in a separate PR. Maybe we should put it as a separate BankEntity since it's from the banks database table.

Reply:
No worries, sounds like a plan